bitnami/kafka/values.yaml

```diff
@@ -825,7 +825,7 @@ controller:
 ##
 broker:
   ## @param broker.replicaCount Number of Kafka broker-only nodes
-  ## Ignore this section if running in Zookeeper mode.
+  ## Ignore this section if you are not running in Zookeeper mode.
```

Comment:
In fact, you can directly remove this line :) `broker` entities are used both in Zookeeper and Kraft modes. This was just a copy-paste duplication of:

https://github.com/bitnami/charts/blob/0f42f4537c1e0bdd63aeede71723da08327ddaf2/bitnami/kafka/values.yaml#L445-L447

Reply:
perfect! updated!